src/src/index.html

```diff
@@ -0,0 +1,122 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    />
+    <title>MOYO header</title>
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="images\logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="#apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#samsung"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#smartphones"
+              class="nav-link"
+            >
```

Comment:
HTML Formatting: Remember to keep your attributes correctly formatted. If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.